.env.sample

```diff
@@ -1,5 +1,8 @@
 SECRET_TOKEN=
 
+EMAIL_ADDRESS=
+EMAIL_PORT=
+EMAIL_DOMAIN=
```

Comment:
let's also add optional `tls` flag.
So we add it to the result hash only when it is set to true.